### PR TITLE
REGISTRY-3216 - Fix for Improvements to notification tab in store and…

### PR DESCRIPTION
**Pull request by mushthaq33**
… publisher

### CODE REVIEW

File: modules/es-extensions/publisher/extensions/app/greg-publisher-defaults/asset.js [hunk 1]
```diff
@@ -39,12 +40,12 @@ asset.renderer = function(ctx) {
     return {
         pageDecorators: {
             sidebarPopulator: function(page) {
-                if (page.meta.pageName === 'details') {
-                    page.isSidebarEnabled = true;
+                if(allowedPagesForSidebar.indexOf(page.meta.pageName)>-1){
```
Comment (daneshk) [inline modules/es-extensions/publisher/extensions/app/greg-publisher-defaults/asset.js]: code formatting issue. please check

File: modules/es-extensions/publisher/extensions/app/greg-publisher-defaults/asset.js [hunk 2]
```diff
@@ -56,12 +57,12 @@ asset.renderer = function(ctx) {
                 }
             },
             notificationPopulator: function(page) {
-                if (page.meta.pageName === 'list' || page.meta.pageName === 'details') {
+                if (allowedPagesForSidebar.indexOf(page.meta.pageName)>-1) {
```
Comment (daneshk) [inline modules/es-extensions/publisher/extensions/app/greg-publisher-defaults/asset.js]: code formatting issue. please check

File: modules/es-extensions/publisher/extensions/app/greg-publisher-defaults/asset.js [hunk 3]
```diff
@@ -56,12 +57,12 @@ asset.renderer = function(ctx) {
...
                     page.notificationsCount = gregAPI.notifications.count();
                 }
             },
             notificationListPopulator: function(page) {
-                if (page.meta.pageName === 'list' || page.meta.pageName === 'details') {
+                if (allowedPagesForSidebar.indexOf(page.meta.pageName)>-1) {
```
Comment (daneshk) [inline modules/es-extensions/publisher/extensions/app/greg-publisher-defaults/asset.js]: code formatting issue. please check

File: modules/es-extensions/publisher/extensions/app/greg-publisher-defaults/asset.js (hunk 1 above)
Comment (mushthaq33) [inline modules/es-extensions/publisher/extensions/app/greg-publisher-defaults/asset.js]: Fixed with https://github.com/wso2/product-greg/pull/850

File: modules/es-extensions/publisher/extensions/app/greg-publisher-defaults/asset.js (hunk 2 above)
Comment (mushthaq33) [inline modules/es-extensions/publisher/extensions/app/greg-publisher-defaults/asset.js]: Fixed with https://github.com/wso2/product-greg/pull/850

File: modules/es-extensions/publisher/extensions/app/greg-publisher-defaults/asset.js (hunk 3 above)
Comment (mushthaq33) [inline modules/es-extensions/publisher/extensions/app/greg-publisher-defaults/asset.js]: Fixed with https://github.com/wso2/product-greg/pull/850

**Pull request merged**